config/manifests/bases/lifecycle-agent.clusterserviceversion.yaml

```diff
@@ -73,4 +74,5 @@ spec:
   maturity: alpha
   provider:
     name: Red Hat
+  replaces: lifecycle-agent.v4.13.0
```

Comment:
Do we plan to release LCA on 4.13?

Reply:
We might use 4.15 values when 4.14 is stated in these changes and for the 'replaces' thing, I am afraid we cannot be congruent as there is no previous value to replace, but the field must exist.

Anyway, all these changes are automatically processed and updated to the right values by midstream builds so the values here shouldn't be relevant.

Reply:
if the value doesn't matter, maybe we don't want to put 4.13 there? People may think it means it was supported on 4.13?

Reply:
`replaces: lifecycle-agent.v0.0.0` ?

Reply:
it will work; at least folks won't get a 4.13 was available as well as @jc-rh commented.